style.css

```diff
@@ -0,0 +1,449 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+}
+body {
+  font-family: "Montserrat", sans-serif;
+}
+
+.header {
+  width: auto;
+  height: 692px;
+  background-color: #fafafa;
+}
+.navbar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 1322px;
+  height: 104px;
+  padding-left: 87px;
+}
+.logo {
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  letter-spacing: 0.1px;
+  color: #252b42;
+  max-width: 130px;
+}
+
+.navbar__menu-item {
```

Comment:
You can group navbar styles together 
`block` `block__element` `block--modifier` or `block__element--modifier`